docs/guides/admin/docs/upgrade.md

```diff
@@ -6,10 +6,62 @@ In case you need information about how to upgrade older versions of Opencast,
 please refer to [older release notes](https://docs.opencast.org).
 
 1. Stop your current Opencast instance
-2. TBA
+2. Replace Opencast with the new version
+3. Review the configuration changes and adjust your configuration accordingly
+4. [Migrate (or rebuild) Elasticsearch index](#migrate-elasticsearch)
 5. Start Opencast
 
-???
----
+Elasticsearch Migration
+-----------------------
 
-TBA
+In Opencast 11, instead of having two separate Elasticsearch indices for the Admin UI and the External API that share
+most of their content, there will be only one index supporting both. Since this index will have a new name, this index
+will either have to be rebuilt from scratch, or you can migrate your Admin UI index, since its structure is completely
+identical to the new index. However, if you don't care about accuracy, you could just keep using the Admin UI index.
```

Comment:
You can't do _nothing_, exactly. If you want to keep using the Admin UI index without migrating it, you would have to change the index name in the config accordingly, as is described in Option 3. Then, the External API will also use the Admin UI index (which is why it's the last option, since this is a bit unclean).

If you don't do anything, Opencast would create an empty index under the new name for both, so then you basically have to do a rebuild.